esp-hal-smartled/src/lib.rs

```diff
@@ -167,10 +164,10 @@ pub const fn buffer_size_rgbw(num_leds: usize) -> usize {
 #[macro_export]
 macro_rules! smart_led_buffer {
     ( $num_leds: expr ) => {
-        [0u32; $crate::buffer_size($num_leds)]
+        [::esp_hal::rmt::PulseCode::default(); $crate::buffer_size($num_leds)]
```

Comment:
Use `PulseCode::end_marker()` otherwise this macro won't work in `const` contexts.